### DISCUSSION TITLE
mssql setup [Aug 30]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#3731] [Discord Threads]
> 🙋‍♂️ **Kalado**

**Comment by o1lab** [comment]
> 🙋‍♂️ **wkw**  
#### i think currently it only takes the first one  
#  
> 🙋‍♂️ **Kalado**  
#### that tracks, thank you. Unfortunately I need all. I also get errors with a few tables :/  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1011532122382671882)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
